cmd/export_test.go

```diff
@@ -0,0 +1,216 @@
+// -----------------------------------------------------------------------------
+// SPDX-FileCopyrightText: Copyright © 2024 bomctl a Series of LF Projects, LLC
+// SPDX-FileName: cmd/export_test.go
+// SPDX-FileType: SOURCE
+// SPDX-License-Identifier: Apache-2.0
+// -----------------------------------------------------------------------------
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+package cmd_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/protobom/protobom/pkg/formats"
+	"github.com/stretchr/testify/require"
+
+	"github.com/bomctl/bomctl/cmd"
+)
+
+func Test_ParseFormat(t *testing.T) {
+	t.Parallel()
+
+	availableEncodings := cmd.EncodingOptions()
+	availableFormats := cmd.FormatOptions()
+
+	for _, format := range availableFormats {
+		formatBase := getFormatBase(format)
+		for _, encoding := range availableEncodings[formatBase] {
+			_, err := cmd.ParseFormat(format, encoding)
+			if err != nil {
+				t.Logf("unexpected error while parsing format: %s with encoding: %s. Err: %v", format, encoding, err)
+				t.FailNow()
+			}
+		}
+	}
+}
+
+func getFormatBase(fmt string) string {
+	if strings.Contains(fmt, formats.CDXFORMAT) {
+		return formats.CDXFORMAT
+	}
+
+	return formats.SPDXFORMAT
+}
+
+func Test_ParseSpecificFormat(t *testing.T) {
```

Comment:
I'd like to stick with the naming convention outlined [here](https://pkg.go.dev/testing#hdr-Examples). Even though the information is for example tests, a lot of projects adopt it for regular tests.

The convention is effectively `Test[StructName_]FunctionName[_OptionalDescriptionOfTestCase]`

```suggestion
func TestParseFormat_Specific(t *testing.T) {
```